### PR TITLE
Updates to guidoism layout

### PR DESCRIPTION
**Pull request by guidoism**

### CODE REVIEW

File: layouts/community/ortho_4x12/guidoism/keymap.c [hunk 1]
```diff
@@ -91,6 +177,14 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
       }
       return false;
       break;
+    case FUNCTION:
```
Comment (drashna) [inline layouts/community/ortho_4x12/guidoism/keymap.c]: You could just use `MO(_FUNCTION)` here.  

And add `#define FUNCTION MO(_FUNCTION)` at the top, too.

File: layouts/community/ortho_4x12/guidoism/readme.md [hunk 2]
```diff
@@ -1,37 +1,56 @@
...
+I've written a script to convert the json file from the
+[QMK Configurator](https://config.qmk.fm) to the C macros
+and a pretty form the comments. 
+
+First import the json file in this directory into the configurator,
+make any changes, export the file and then run this:
+
+    mv ~/Downloads/guidoism.json layouts/community/ortho_4x12/guidoism/ && python3 layouts/community/ortho_4x12/guidoism/generate_c.py && make planck:guidoism:dfu
+
+TODO: Move running of generate_c.py to makefile so I just need to run `make planck:guidoism:dfu`
```
Comment (drashna) [inline layouts/community/ortho_4x12/guidoism/readme.md]: I think you can use the `rules.mk` to do this, actually.  

I'm not entirely sure how, but, if you check the chibos.mk and avr.mk files in tmk_core, that should give you an idea on how.

File: layouts/community/ortho_4x12/guidoism/keymap.c (hunk 1 above)
Comment (guidoism) [reply layouts/community/ortho_4x12/guidoism/keymap.c]: Done.

File: layouts/community/ortho_4x12/guidoism/readme.md (hunk 2 above)
Comment (guidoism) [reply layouts/community/ortho_4x12/guidoism/readme.md]: Thank you for the suggestion, I'll take a look at this when I have time.

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**